### PR TITLE
machinectl: add page

### PR DESCRIPTION
**Pull request by rajveer43**
#10191

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/machinectl.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Machinectl
```
Comment (kbdharun) [inline pages/linux/machinectl.md:1]:
```suggestion
# machinectl
```

Note: We don't capitalize the words in page headings generally (as we keep it the same as case as the command)

File: pages/linux/machinectl.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+> Machinectl is a command-line utility for managing Linux containers, particularly on systems that use the systemd init system.
```
Comment (kbdharun) [inline pages/linux/machinectl.md:3]:
```suggestion
> Utility to manage Linux containers, particularly on systems that use the `systemd init` system.
```
Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#descriptions

File: pages/linux/machinectl.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+> It provides a powerful and convenient way to create, start, stop, and manage container instances, referred to as "machines" in systemd terminology.
```
Comment (kbdharun) [inline pages/linux/machinectl.md:4]:
```suggestion
> It provides a powerful and convenient way to create, start, stop, and manage container instances, referred to as machines in `systemd` terminology.
> More information: <https://www.freedesktop.org/software/systemd/man/machinectl.html>.
```
We can add a link to the man page here.

File: pages/linux/machinectl.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Create a Machine:
+
+`sudo machinectl create my-container`
```
Comment (kbdharun) [inline pages/linux/machinectl.md:8]:
```suggestion
- Create a container:

`sudo machinectl create {{container-name}}`
```

File: pages/linux/machinectl.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- You can start a container with the following command:
+
+`sudo machinectl start my-container`
+
+- To stop a running container, use the following command:
+
+`sudo machinectl stop my-container`
```
Comment (kbdharun) [inline pages/linux/machinectl.md:16]:
```suggestion
- Start a container:

`sudo machinectl start {{container-name}}`

- Stop a running container:

`sudo machinectl stop {{container-name}}`
```

File: pages/linux/machinectl.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+- To see a list of running containers, execute:
```
Comment (kbdharun) [inline pages/linux/machinectl.md:18]:
```suggestion
- Display a list of running containers:
```

File: pages/linux/machinectl.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+`machinectl list`
+
+- To log into a container for interactive use, use:
```
Comment (kbdharun) [inline pages/linux/machinectl.md:22]:
```suggestion
- Interactively log into the container shell:
```

Comment (kbdharun) [inline pages/linux/machinectl.md:22]:
```suggestion
- Interactively log into the container shell:
```

File: pages/linux/machinectl.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+`sudo machinectl shell my-container`
```
Comment (kbdharun) [inline pages/linux/machinectl.md:24]:
```suggestion
- Interactively log into the container shell:

`sudo machinectl shell {{container-name}}`
```

File: pages/linux/machinectl.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+- Machinectl is typically available on systems that support systemd. If not installed, you can usually obtain it via your distribution's package manager. For example, on Debian/Ubuntu:
+
+`sudo apt-get install systemd-container`
```
Comment (kbdharun) [inline pages/linux/machinectl.md:28]:
```suggestion
```

We don't support having installation commands for any distribution specific packages in tldr.

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

Note: We wrap placeholders with `{{}}`, for our clients to highlight it in a different style/color.

**Review by kbdharun — APPROVED** [review]

File: pages/linux/machinectl.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
+# machinectl
+
+> Utility to manage Linux containers, particularly on systems that use the `systemd init` system.
+> It provides a powerful and convenient way to create, start, stop, and manage container instances, referred to as machines in `systemd` terminology.
```
Comment (acuteenvy) [inline pages/linux/machinectl.md:4]:
```suggestion
> Control the systemd machine manager.
> Execute operations on virtual machines, containers and images.
```

File: pages/linux/machinectl.md [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+> More information: <https://www.freedesktop.org/software/systemd/man/machinectl.html>.
+
+- Create a container:
+
+`sudo machinectl create {{container-name}}`
+
+- Start a container:
+
+`sudo machinectl start {{container-name}}`
```
Comment (acuteenvy) [inline pages/linux/machinectl.md:13]:
```suggestion
- Start a container as a service using `systemd-nspawn`:

`sudo machinectl start {{container_name}}`
```

File: pages/linux/machinectl.md [hunk 12]
```diff
@@ -0,0 +1,25 @@
+# machinectl
+
+> Utility to manage Linux containers, particularly on systems that use the `systemd init` system.
+> It provides a powerful and convenient way to create, start, stop, and manage container instances, referred to as machines in `systemd` terminology.
+> More information: <https://www.freedesktop.org/software/systemd/man/machinectl.html>.
+
+- Create a container:
+
+`sudo machinectl create {{container-name}}`
```
Comment (acuteenvy) [inline pages/linux/machinectl.md:9]: This doesn't seem to work:
```
# machinectl create container_name
Unknown command verb create.
```

File: pages/linux/machinectl.md [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+
+- Start a container:
+
+`sudo machinectl start {{container-name}}`
+
+- Stop a running container:
+
+`sudo machinectl stop {{container-name}}`
```
Comment (acuteenvy) [inline pages/linux/machinectl.md:17]:
```suggestion
- Stop a running machine:

`sudo machinectl stop {{machine_name}}`
```

File: pages/linux/machinectl.md [hunk 14]
```diff
@@ -0,0 +1,25 @@
...
+
+- Display a list of running containers:
+
+`machinectl list`
+
+- Interactively log into the container shell:
+
+`sudo machinectl shell {{container-name}}`
```
Comment (acuteenvy) [inline pages/linux/machinectl.md:25]:
```suggestion
- Display a list of running machines:

`machinectl list`

- Open an interactive shell inside the machine:

`sudo machinectl shell {{machine_name}}`
```

File: pages/linux/machinectl.md (hunk 12 above)
Comment (kbdharun) [reply pages/linux/machinectl.md:9]: I too can confirm it doesn't work.

```zsh
➜  ~ machinectl create
Unknown command verb create.
➜  ~ sudo machinectl create test
Place your right index finger on the fingerprint reader
Unknown command verb create.
```

Version:

```zsh
➜  ~ machinectl --version
systemd 253 (253.10-1.fc38)
+PAM +AUDIT +SELINUX -APPARMOR +IMA +SMACK +SECCOMP -GCRYPT +GNUTLS +OPENSSL +ACL +BLKID +CURL +ELFUTILS +FIDO2 +IDN2 -IDN -IPTC +KMOD +LIBCRYPTSETUP +LIBFDISK +PCRE2 +PWQUALITY +P11KIT +QRENCODE +TPM2 +BZIP2 +LZ4 +XZ +ZLIB +ZSTD +BPF_FRAMEWORK +XKBCOMMON +UTMP +SYSVINIT default-hierarchy=unified
```

File: pages/linux/machinectl.md [hunk 15]
```diff
@@ -0,0 +1,25 @@
+# machinectl
+
+> Control the systemd machine manager.
+> Execute operations on virtual machines, containers and images.
+> More information: <https://www.freedesktop.org/software/systemd/man/machinectl.html>.
+
+- Create a container:
+
+`sudo machinectl create {{container-name}}`
```
Comment (kbdharun) [inline pages/linux/machinectl.md:9]:
```suggestion
```

**Review by kbdharun — APPROVED** [review]

**Review by gutjuri — APPROVED** [review]

**Pull request merged**